tmt/tests/system-reinstall/test-system-reinstall-bootc.sh

```diff
@@ -0,0 +1,140 @@
+            cp -r /usr/local/bin .
+            ;;
+        "fedora")
+            BASE="quay.io/fedora/fedora-bootc:${VERSION_ID}"
+            TEST_OS="${ID}-${VERSION_ID}"
+            if [[ "$VERSION_ID" == 44 ]]; then
+                TEST_OS="${ID}-rawhide"
+            fi
+            # Some rhts-*, rstrnt-* and tmt-* commands are in /usr/local/bin
+            cp -r /usr/local/bin .
+            ;;
+        "rhel")
+            # OSCI gating only
+            BASE="images.paas.redhat.com/bootc/rhel-bootc:latest-${VERSION_ID}"
+            # Some rhts-*, rstrnt-* and tmt-* commands are in /var/lib/tmt/scripts
+            cp -r /var/lib/tmt/scripts ./bin
+
+            CURRENT_COMPOSE_ID=$(skopeo inspect --no-tags --retry-times=5 --tls-verify=false "docker://${BASE}" | jq -r '.Labels."redhat.compose-id"')
+
+            if [[ -n ${CURRENT_COMPOSE_ID} ]]; then
+                if [[ ${CURRENT_COMPOSE_ID} == *-updates-* ]]; then
+                    BATCH_COMPOSE="updates/"
+                else
+                    BATCH_COMPOSE=""
+                fi
+            else
+                BATCH_COMPOSE="updates/"
+                CURRENT_COMPOSE_ID=latest-RHEL-$VERSION_ID
+            fi
+
+            # use latest compose if specific compose is not accessible
+            RC=$(curl -skIw '%{http_code}' -o /dev/null "http://${NIGHTLY_COMPOSE_SITE}/rhel-${VERSION_ID%%.*}/nightly/${BATCH_COMPOSE}RHEL-${VERSION_ID%%.*}/${CURRENT_COMPOSE_ID}/STATUS")
+            if [[ $RC != "200" ]]; then
+                CURRENT_COMPOSE_ID=latest-RHEL-${VERSION_ID%%}
+            fi
+
+            # generate rhel repo
+            tee "rhel.repo" >/dev/null <<REPOEOF
+[rhel-baseos]
+name=baseos
+baseurl=http://${NIGHTLY_COMPOSE_SITE}/rhel-${VERSION_ID%%.*}/nightly/${BATCH_COMPOSE}RHEL-${VERSION_ID%%.*}/${CURRENT_COMPOSE_ID}/compose/BaseOS/${ARCH}/os/
+enabled=1
+gpgcheck=0
+
+[rhel-appstream]
+name=appstream
+baseurl=http://${NIGHTLY_COMPOSE_SITE}/rhel-${VERSION_ID%%.*}/nightly/${BATCH_COMPOSE}RHEL-${VERSION_ID%%.*}/${CURRENT_COMPOSE_ID}/compose/AppStream/${ARCH}/os/
+enabled=1
+gpgcheck=0
+REPOEOF
+            cp rhel.repo /etc/yum.repos.d
+            ;;
+    esac
+
+    # Fedora CI: https://github.com/fedora-ci/dist-git-pipeline/blob/master/Jenkinsfile#L145
+    # OSCI: https://gitlab.cee.redhat.com/osci-pipelines/dist-git-pipeline/-/blob/master/Jenkinsfile?ref_type=heads#L93
+    if [[ -v KOJI_TASK_ID ]] || [[ -v CI_KOJI_TASK_ID ]]; then
+        # Just left those ls commands here to ring the bell for me when something changed
+        echo "$TMT_SOURCE_DIR"
+        ls -al "$TMT_SOURCE_DIR"
+        ls -al "$TMT_SOURCE_DIR/SRPMS"
+        ls -al /etc/yum.repos.d
+        cat /etc/yum.repos.d/test-artifacts.repo
+        ls -al /var/share/test-artifacts
+        # copy bootc rpm repo into image building root
+        cp /etc/yum.repos.d/test-artifacts.repo ../../hack/
+    # Packit
+    else
+        tee "test-artifacts.repo" <<PACKITEOF
```

Comment:
Hmmm won't tmt+testing farm inject something like this by default into `/etc/yum.repos.d`?